### ISSUE TITLE
bucketIterator still requires device number specified.

### ISSUE THREAD

**Issue by ysangj** [pytorch/text#327]
`train_iter, test_iter = data.BucketIterator.splits(
    (train, test), batch_size=batch_size, shuffle=True, repeat=False, device = -1)`

I am using pytorch 0.4 in Mac OS. I thought I can specify the machine by 'cpu', instead of -1.
Will this be updated soon?

**Comment by zhangguanheng66** [comment]
@ysangj do you still experience such issue. Please reopen it if so. Thanks.
